backend/Squirrel.ConsoleApp.Models/Models/DbType.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Squirrel.ConsoleApp.Models.Models;
+
+public enum DbType
```

Comment:
we also have `DbType` in `Systems` it's probably better to rename it as `DatabseType` to avoid misunderstanding

Reply:
﻿we also have `DbEngine` in `Squirrel.Core.DAL.Enums`

```
public enum DbEngine
{
    MsSqlServer = 1,
    PostgreSql
}
```

Reply:
or to `DbEngine`